pages/desktop/user.py

```diff
@@ -119,8 +120,19 @@ def click_update_account(self):
         self.selenium.find_element(*self._update_account_locator).click()
         WebDriverWait(self.selenium, self.timeout).until(lambda s: self.update_message == "Profile Updated")
 
-    def change_hide_email_state(self):
-        self.selenium.find_element(*self._hide_email_checkbox).click()
+    def hide_email_address(self):
+        email_checkbox = self.selenium.find_element(*self._email_checkbox)
+        WebDriverWait(self.selenium, self.timeout).until(expected_conditions.visibility_of(email_checkbox));
+        if not email_checkbox.is_selected():
+                self.selenium.find_element(*self._email_checkbox).click()
```

Comment:
The point is that you should have an expectation of the state before the test runs. If you expect that the email will already be hidden, then you shouldn't be calling `hide_email_address` in the test. A good test should always have a known state at the beginning of the test, and a known state at any point. Are you saying that we cannot know the initial state? That sometimes it will start hidden and sometimes it will not? If that is the case then I suggest that there is a problem with the test.